docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.8'
+name: sandbox
+
+services:
+  algod:
+    container_name: sandbox_algod
+    image: makerxau/algorand-sandbox-dev:latest
+    ports:
+      - "4001:4001"
+      - "4002:4002"
+      - "9392:9392"
+
+  indexer:
+    container_name: sandbox_indexer
+    image: makerxau/algorand-indexer-dev:latest
+    ports:
+      - "8980:8980"
+    restart: unless-stopped
+    environment:
+      ALGOD_HOST: algod
+      POSTGRES_HOST: indexer-db
+      POSTGRES_PORT: 5432
+      POSTGRES_USER: algorand
+      POSTGRES_PASSWORD: algorand
+      POSTGRES_DB: indexer_db
+    depends_on:
+      - indexer-db
+      - algod
+
+  indexer-db:
+    container_name: sandbox_postgres
+    image: postgres:14-alpine
+    user: postgres
+    environment:
+      POSTGRES_USER: algorand
+      POSTGRES_PASSWORD: algorand
+      POSTGRES_DB: indexer_db
```

Comment:
Could we remove the indexer requirements for this? Would that make it faster? I'm not sure we _need_ them for testing

Reply:
Good question! The test will pass without it it appears. I put indexer in because it originally used sandbox so wanted to mimic that. It doesn't appear to speed things up much, but let's remove it for now since it doesn't appear to be required.

Reply:
done ✅